app/models/ems_refresh/refreshers/container_refresher_mixin.rb

```diff
@@ -0,0 +1,38 @@
+module EmsRefresh
+  module Refreshers
+    module ContainerRefresherMixin
+      def kubernetes_fetch_inventory(client)
+        fetch_entities(client, KUBERNETES_ENTITIES)
+      end
+
+      def openshift_fetch_inventory(client)
+        fetch_entities(client, OPENSHIFT_ENTITIES)
+      end
```

Comment:
This is definitely better than having container-specific things in the `ems_refresh_mixin`.  

Though, I'm still not sure I understand why this needs to be in a mixin which then talks about specific container types.  I mean, I understand that the kubernetes stuff is shared between Kubernetes and Openshift.  But, that goes back to my original point that I think this could probably be handled through inheritance or through some builder pattern, since Openshift == Kubernetes + Extra Stuff.  But, @simon3z and I have a difference of opinion there.

I won't hold up the PR because of it.  But, I'd still like to talk about it eventually.  Maybe we can look at refactoring the entire refresher procedure in general and find a better way to handle this in the process.